examples/02_first_level_models/plot_fiac_analysis.py

```diff
@@ -1,9 +1,9 @@
-"""
-Simple example of GLM fitting in fMRI
-======================================
+"""Simple example of two-session fMRI model fitting
```

Comment:
May be "Comparing two-session fMRI model fitting seperately to fixed-effects analysis".

I am trying to avoid Sentence starting with Simple example. We can write that inside the example for instance.